ckanext/security/constants.py

```diff
@@ -0,0 +1 @@
+PLUGIN_EXTRAS_TABULIST_KEY = 'password_tabulist'
```

Comment:
This reads more clearly if "tabu" and "list" are separated into two words, so that it doesn't resemble table-related words like "tabular". Or change to "blacklist".